homeassistant/components/teslemetry/entity.py

```diff
@@ -148,6 +151,21 @@ def __init__(
         super().__init__(data.info_coordinator, data.api, key)
 
 
+class TeslemetryEnergyHistoryEntity(TeslemetryEntity):
+    """Parent class for Teslemetry Energy History Entities."""
+
+    def __init__(
+        self,
+        data: TeslemetryEnergyData,
+        key: str,
+    ) -> None:
+        """Initialize common aspects of a Teslemetry Energy Site Info entity."""
+        self._attr_unique_id = f"{data.id}-{key}"
+        self._attr_device_info = data.device
+
+        super().__init__(data.history_coordinator, data.api, key)
```

Comment:
No immediate plans to. I like keeping it consistent with everything else but I guess you're going to say there shouldn't be a base case since it won't get reused ever?